src/JobTracker/chatbot.py

```diff
@@ -26,6 +27,54 @@ def get_content(self, info):
         pass
 
 
+class Llama(ChatBot):
+    def __init__(self) -> None:
+        super().__init__()
+        self.url = "http://127.0.0.1:11434/api/generate"
```

Comment:
Don't hardcode the URL here, put it in `config.py`, User may use other port for it.